packages/lldb/utils-TableGen-LLDBTableGen.cpp.patch.beforehostbuild

```diff
@@ -0,0 +1,14 @@
+diff --git a/utils/TableGen/LLDBTableGen.cpp b/utils/TableGen/LLDBTableGen.cpp
+index abb6589f0ca6..aa29873d16a2 100644
+--- a/utils/TableGen/LLDBTableGen.cpp
++++ b/utils/TableGen/LLDBTableGen.cpp
+@@ -42,6 +42,9 @@ static cl::opt<ActionType> Action(
+                clEnumValN(GenPropertyEnumDefs, "gen-lldb-property-enum-defs",
+                           "Generate lldb property enum definitions")));
+ 
++static cl::opt<bool>
++WriteIfChanged("write-if-changed", cl::desc("Only write output if it changed, does nothing"));
```

Comment:
The LLVM CMake config tries to pass this flag to lldb-tablegen, but it doesn't have it, so I just copied this from llvm-tablegen and had it do nothing.

Reply:
The reason it didn't have this flag is because the host lldb-tblgen was linked against the system LLVM 9 then, whereas I'm now linking against LLVM 10 built in the previous step so it works and this patch wasn't needed anymore.